api/streams.go

```diff
@@ -59,8 +59,21 @@ type PubAck struct {
 	Duplicate bool   `json:"duplicate,omitempty"`
 }
 
+// ApiPagedRequest includes parameters allowing specific pages to be requests from APIs responding with ApiPaged
+type ApiPagedRequest struct {
```

Comment:
See JSApiIterableRequest

Reply:
Hadn't noticed I was re-inventing it, thanks